week1/task3/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿// See https://aka.ms/new-console-template for more information
+int[] arr = new[] { 271, -3, 1, 14, -100, 13, 2, 1, -8, -59, -1852, 41, 5 };
+int sumOfNegativeNumbers = 0;
+int multipleOfPositiveNumbers = 1;
+int[] GetResult(int[] arr)
+{
+    for (int i = 0; i < arr.Length; i++)
+    {
+        if (arr[i] < 0)
+        {
+            sumOfNegativeNumbers += arr[i];
+        }
+        else if (arr[i] >= 0)
+        {
+            multipleOfPositiveNumbers = multipleOfPositiveNumbers * arr[i];
```

Comment:
Very good!!
You can also write this line like this:
multipleOfPositiveNumbers  *= arr[i];

Reply:
yes, I will replace it.